html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm

```diff
@@ -48,7 +48,7 @@ The captival portal block
 
 has_block 'captive_portal' =>
   (
-    render_list => [qw(logo redirecturl always_use_redirecturl nbregpages block_interval sms_pin_retry_limit sms_request_limit login_attempt_limit)],
+    render_list => [qw(logo redirecturl always_use_redirecturl preregistration nbregpages block_interval sms_pin_retry_limit sms_request_limit login_attempt_limit)],
```

Comment:
I think it should be on the 'Settings' tab of the portal profile more than on the 'Captive Portal' one since it is a fairly "important" setting in the sense of, it disables local authentication if enabled...